package/com.unity.formats.usd/Runtime/Scripts/IO/SanitizedSamples.cs

```diff
@@ -9,6 +9,31 @@
 [assembly: InternalsVisibleToAttribute("Unity.Formats.USD.Tests")]
 namespace Unity.Formats.USD
 {
+    /// <summary>
+    /// When a prim is sparsely animated only dynamic properties are read into the Sample.
+    /// The ConversionSate purpose is to hold the data (static properties or computed data) to ensure proper conversion.
+    /// </summary>
+    public interface IRestorable
+    {
+        /// <summary>
+        /// Returns true if the internal data has been restored from State data
+        /// </summary>
+        public bool IsRestored();
```

Comment:
It would be awesome if we could remove this bool - for some reason, it triggers read flags for me, in terms of creating future bugs.